k8s/webapp/values.yaml

```diff
@@ -169,7 +169,19 @@ nodeSelector: {}
 
 tolerations: []
 
-affinity: {}
+affinity:
```

Comment:
maybe add it into the `webapi.yaml` and `identity.yaml` values in k8s? As adding this into webapp base chart directly may be replaced accidently later on using values override.